### PR TITLE
Code cleanup, tooltips in grid view and close button background the same as navbar

### PR DESCRIPTION
**Pull request by Iliannnn**
- Added tooltips when hovering over a result in grid view
- Made the background color of the close button in the grid view the same as Smartschool's navbar (also works with custom and dark theme)
- Code cleanup

### CODE REVIEW

File: BetterResults.js
Comment (EbbDrop) [inline BetterResults.js:223]: Can you change these to `rem` instead of `px` units. You also left a second margin-left at the top.

**Review by EbbDrop — CHANGES_REQUESTED** [review]
Thank you for making this. I definitely want to merge the pop ups but I'm not sure about the color changes. Could you split those into a separate PR.

There also seems to be a problem with the disclaimer:
Screenshot_2022-11-30_21-52-37

**Comment by Iliannnn** [comment]
> Thank you for making this. I definitely want to merge the pop ups but I'm not sure about the color changes. Could you split those into a separate PR.
> 
> There also seems to be a problem with the disclaimer: Screenshot_2022-11-30_21-52-37

jGWqP2
Fixed that now. Will split the features now.

File: Changelog.txt
Comment (EbbDrop) [inline Changelog.txt:4]:
```suggestion
- Added tooltips when hovering over a result in grid view
- Code cleanup
```

You forgot to remove this

**Review by EbbDrop — CHANGES_REQUESTED** [review]
Just forgot to update the change log. This looks good to merge after that

**Comment by Iliannnn** [comment]
> Just forgot to update the change log. This looks good to merge after that

Updated

**Comment by EbbDrop** [comment]
Thanks for the update. It will take a few days before google approves and pushes the updates to everybody.

**Pull request merged**
